.github/workflows/ci.yaml

```diff
@@ -28,31 +27,79 @@ permissions:
   pull-requests: write
 
 jobs:
-  build_and_deploy:
+  build:
     if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
```

Comment:
this condition check is reused for some later job.
how about storing it to variable?

Reply:
I tried, and there doesnt seem to be a context available, where we can set and access such variable. We could set it in **env** context, but cannot use it. For job conditionals, only **github** context is available, which is read only :(